## 3) Calculating quantiles with qnorm

Mr. Gittu also wants know what exam score corresponds to the 90th percentile. Using `qnorm`, he can easily find out what test score students need to to be in the top 10% of the class. Using the parameter `lower_tail = False`, Mr. Gittu can also identify the test score that 90% of the class will score above.

In [19]:
from pystats.qnorm import qnorm

quantile1 = qnorm(p=0.9, mean=70, sd=10)
print(round(quantile1, 2))

82.82


To be in the top 10% of the class, Mr. Gittus' students will need to score at least 82.82% on the test.

In [15]:
quantile2 = qnorm(p=0.9, mean=70, sd=10, lower_tail = False)
print(round(quantile2, 2))

57.18


Conversely, 90% of Mr. Gittus' students should score above 57.18% on the test.